edit-post/components/layout/index.js

```diff
@@ -35,21 +35,20 @@ import {
 	isFeatureActive,
 	getOpenedGeneralSidebar,
 	isPublishSidebarOpened,
-	getActivePlugin,
 	getMetaBoxes,
 	hasMetaBoxes,
 	isSavingMetaBoxes,
 } from '../../store/selectors';
 import { closePublishSidebar } from '../../store/actions';
-import PluginsPanel from '../../components/plugins-panel/index.js';
-import { getSidebarSettings } from '../../api/sidebar';
+import { PluginSidebarSlot } from '../../components/plugin-sidebar/index.js';
+import { PluginFills } from '../../api/plugin';
 
 function GeneralSidebar( { openedGeneralSidebar } ) {
 	switch ( openedGeneralSidebar ) {
 		case 'editor':
 			return <Sidebar />;
 		case 'plugin':
-			return <PluginsPanel />;
+			return <PluginSidebarSlot />;
```

Comment:
It would be nice to avoid using the `slot` part in the name of the component. It should be known only internally.